GagarinDmitry/src/Services/Catalog/Catalog.Application/Models/GetSessionModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Catalog.Domain.Entities.Types;
+
+namespace Catalog.Application.Models
+{
+    public class GetSessionModel
+    {
```

Comment:
Why do you use such name for this class? As for me, such name is suitable for method, not for class)